public/wetty/hterm_all.js

```diff
@@ -12185,10 +12185,6 @@ hterm.Terminal.prototype.copySelectionToClipboard = function() {
     this.copyStringToClipboard(text);
 };
 
-hterm.Terminal.prototype.overlaySize = function() {
-  this.showOverlay(this.screenSize.width + 'x' + this.screenSize.height);
-};
-
```

Comment:
Maybe we can leave this part in case we need it for some reason or does that break it?

Reply:
Good call. Now added back, it will not break it.